### ISSUE TITLE
Excessive expansion of -o-transition values break Opera 12

### ISSUE THREAD

**Issue by 5lava** [postcss/autoprefixer#642]
```css
.example {
    transition: opacity 1s ease, transform 1 ease;
}
```
```
browsers = "> 0%"
```
Result contains following lines:
```
-webkit-transition: opacity 1s ease-out, -webkit-transform 1s ease-out;
   -moz-transition: opacity 1s ease-out, transform 1s ease-out, -moz-transform 1s ease-out;
     -o-transition: opacity 1s ease-out, transform 1s ease-out, -o-transform 1s ease-out;
```
Pay attention to line 3. I don't know if `transform 1 ease` was added intentionally (it was also added to `moz` in line 2, but not to `webkit` in line 1), but in Opera 12 it actually breaks entire `-o-transition` declaration, leading to no transition at all (i.e. transition is not applied to neither `transform` nor `opacity`). What actually works is:
```
-o-transition: opacity 1s ease, -o-transform 1 ease;
```

**Comment by ai** [comment]
Done d06c0a8

Please check out before release (it will be in 6.4)

**Comment by ai** [comment]
Released in 6.4.
